### ISSUE TITLE
Service account/rolebindings lack labels even though cluster specific.

### ISSUE THREAD

**Issue by GrahamDumpleton** [strimzi/strimzi-kafka-operator#1305]
Most objects created have labels. Eg:.

```
Labels:             strimzi.io/cluster=my-cluster
                    strimzi.io/kind=Kafka
                    strimzi.io/name=my-cluster-zookeeper
```

In mode where Kafka operator watches all projects using ``*`` at least, the service accounts and role bindings created in a project, do not have the labels, even though they are named specific to the cluster created.

```
$ oc get serviceaccount,rolebinding -o name
serviceaccount/builder
serviceaccount/default
serviceaccount/deployer
serviceaccount/my-cluster-kafka
rolebinding.authorization.openshift.io/admin
rolebinding.authorization.openshift.io/system:deployers
rolebinding.authorization.openshift.io/system:image-builders
rolebinding.authorization.openshift.io/system:image-pullers
rolebinding.authorization.openshift.io/try-kafka-hhsj7-admin
rolebinding.authorization.openshift.io/try-kafka-hhsj7-extra
rolebinding.authorization.openshift.io/try-kafka-hub-admin

$ oc describe serviceaccount/my-cluster-kafka
Name:                my-cluster-kafka
Namespace:           try-kafka-hhsj7
Labels:              <none>
Annotations:         <none>
Image pull secrets:  my-cluster-kafka-dockercfg-b5cdt
Mountable secrets:   my-cluster-kafka-dockercfg-b5cdt
                     my-cluster-kafka-token-lvpq6
Tokens:              my-cluster-kafka-token-lvpq6
                     my-cluster-kafka-token-tw445
Events:              <none>
```

Would be nice if labelling was consistent and applied to these as well. Thus can use label selector and do:

```
oc get all,configmap,serviceaccount,rolebinding --selector strimzi.io/cluster=my-cluster
```

and pick up everything.

**Comment by scholzj** [comment]
Good point. We will have a look at it. Thanks for raising this issue.

**Comment by scholzj** [comment]
This should be closed by #1346. I will close this issue. Thanks for raising it.
